lib/internal/util.js

```diff
@@ -186,3 +187,24 @@ exports.toLength = function toLength(argument) {
   const len = toInteger(argument);
   return len <= 0 ? 0 : Math.min(len, Number.MAX_SAFE_INTEGER);
 };
+
+let signalsToNamesMapping;
+function getSignalsToNamesMapping() {
+  if (signalsToNamesMapping === undefined) {
```

Comment:
Could you reverse the `if` logic and return if `signalsToNamesMapping` exists. That can save some indentation on the following lines.